.travis.yml

```diff
@@ -30,10 +31,10 @@ addons:
     - libssl-dev
   coverity_scan:
     project:
-      name: "01org/tpm2-tss"
+      name: "01org/TPM2.0-TSS"
       description: Build submitted via Travis-CI
     notification_email: philip.b.tricca@intel.com
-    build_command_prepend: "make clean"
+    build_command_prepend: "./bootstrap && mkdir cov-build && pushd cov-build && ../configure"
```

Comment:
no need to mkdir here. Just ./bootstrap && ./configure
and modify script: with this
'''
script:
  - if [[ "${COVERITY_SCAN_BRANCH}" == 1 ]];
      then
        echo "Don't build on coverty_scan branch.";
        exit 0;
    fi
  - make
'''